chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts

```diff
@@ -110,7 +112,12 @@ export class Pick3DrawDateInfoSection implements OnInit {
             } else {
                 this.getPastWinningDrawingNumber(this.data.getDrawState(), pick3DrawTime.getDateTime(), pick3DrawTime.getType());
             }
-            this.showCountDownToDrawing = true;
+            const path = location.pathname;
```

Comment:
Combine duplicate code into single method and reuse the method here.

Reply:
Reduced to single method and reused.